packages/z2s/src/compiler.output.test.ts

```diff
@@ -33,7 +34,9 @@ const user = table('user')
   .columns({
     id: string(),
     name: string(),
+    nameArray: json<string[]>(),
```

Comment:
is an array of json possible in PG? E.g., `json<json[]>`. Granted it seems like an insane edge case.

Reply:
From zero's point of view it is just json. I haven't tested if that maps cleanly... Or at least I don't remember